extensions/host/internal.m

```diff
@@ -557,13 +557,11 @@ static int hs_idleTime(lua_State *L) {
         CFNumberGetValue((CFNumberRef)idle, kCFNumberSInt64Type, &time) ;
     } else {
         IOObjectRelease(ioIterator) ;
-        CFRelease(idle) ;
         CFRelease(properties) ;
         return luaL_error(L, "Unsupported type %d for HIDIdleTime", type) ;
     }
 
     IOObjectRelease(ioIterator) ;
-    CFRelease(idle) ;
```

Comment:
ditto.

Reply:
Same source as above, but I'm a little surprised it never caused an issue during my testing since this would get called everytime a number is returned... however, I agree that one would expect it to follow the Get rule.

 `IORegistryEntryCreateCFProperties` creates the dictionary, and the docs don't say whether or not that uses the default value callback functions when doing so... I guess try we try it and see.